themes/codefor-theme/layouts/partials/simple_footer.html

```diff
@@ -20,6 +20,9 @@ <h4 class="mb-4">Werkzeugkoffer</h4>
               <li>
                 <a class="text-dark no-underline" href="https://opendatahandbook.org" target="_blank" rel="noopener noreferrer" >opendatahandbook.org</a>
               </li>
+              <li>
+                <a class="text-dark no-underline" href="https://github.com/okfde/hackathon-leitfaden/releases/download/v1.0.0/hackathon-leitfaden-v1.0.0.pdf" target="_blank" rel="noopener noreferrer" > Handbuch für Hackathons in Verwaltungen</a>
```

Comment:
Funny whitespace. Also isn't the `target="blank" rel="noopener noreferrer"` handeled by hugo now?

```suggestion
                <a class="text-dark no-underline" href="https://github.com/okfde/hackathon-leitfaden/releases/download/v1.0.0/hackathon-leitfaden-v1.0.0.pdf" target="_blank" rel="noopener noreferrer">Handbuch für Hackathons in Verwaltungen</a>
```

Reply:
- The whitespace is minified on deployment

- The links are only handled by hugo if they are Markdown links, not plain html links